components/actiongroup/index.css

```diff
@@ -10,11 +10,6 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-/* hack: use var so it's not reported as unused */
```

Comment:
This update means we no longer need to include these hacks - unused variables and passthroughs will be allowed and only show up as informational in linting.